winui/Cartesian-Charts/Getting-Started.md

```diff
@@ -118,38 +120,41 @@ N> Add namespace of `ViewModel` class to your XAML Page if you prefer to set `Da
     mc:Ignorable="d" Height="350" Width="525"
     Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
 
-    <Window.DataContext>
-        <model:ViewModel></local:ViewModel>
-    </Window.DataContext>
+    <Grid x:Name="grid">
+	    <Grid.DataContext>
+	        <model:ViewModel/>
+	    </Grid.DataContext>
+	</Grid>	
+	
 </Window>
 
 {% endhighlight %}
 
 {% highlight C# %} 
 
-this.DataContext = new ViewModel();
+grid.DataContext = new ViewModel();
 
 {% endhighlight %}
 
 {% endtabs %} 
 
 ## Initialize Chart Axis
 
-[ChartAxis](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.ChartAxis.html) is used to locate the data points inside the chart area. The [PrimaryAxis](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.SfCartesianChart.html#Syncfusion_UI_Xaml_Charts_SfCartesianChart_PrimaryAxis) and [SecondaryAxis](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.SfCartesianChart.html#Syncfusion_UI_Xaml_Charts_SfCartesianChart_SecondaryAxis) properties of the chart is used to initialize the axis for the chart.
+[ChartAxis](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.ChartAxis.html) is used to locate the data points inside the chart area. The [XAxes]() and [YAxes]() collection of the chart is used to initialize the axis for the chart.
```

Comment:
The [ChartAxis](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.ChartAxis.html) is used to locate the data points inside the chart area. The [XAxes]() and [YAxes]() collection of the chart is used to initialize the axis for the chart.